content/en/service_management/workflows/trigger.md

```diff
@@ -152,6 +152,24 @@ You can manually start a workflow from a Cloud SIEM Security Signal panel.
 
 For additional examples of security workflows you can automate, see [Automate Security Workflows with Workflow Automation][4].
 
+## GitHub triggers
+
+<div class="alert alert-info"><strong>Note</strong>: Your GitHub account must have permission to create webhooks for a GitHub repo to use this feature.</div>
```

Comment:
Technically GitHub webhooks can be triggered from more than just a repositroy. See [here](https://docs.github.com/en/webhooks/types-of-webhooks) for the different webhook types.  Regardless it is a good callout that you must have to be the owner or admin privileges to create a Github webhook (depends on the webhook type). Not sure if it is worth being explicit here.

Reply:
Okay, I can take out the bit about repo